web/src/components/IdeViewer/IdeViewer.js

```diff
@@ -126,10 +139,17 @@ const IdeViewer = () => {
             />
             <ambientLight />
             <pointLight position={[15, 5, 10]} />
-            <Box position={[0, 0, 0]} size={10} color="orange" />
-            <Box position={[0, 5, 0]} size={1} color="cyan" />
-            <Box position={[0, 0, 5]} size={1} color="magenta" />
-            <Box position={[5, 0, 0]} size={1} color="hotpink" />
+            <Box position={[0, 4.95, 0]} size={[0.1, 10, 0.1]} color="cyan" />
+            <Box
+              position={[0, 0, -5.05]}
+              size={[0.1, 0.1, 10]}
+              color="orange"
```

Comment:
Are you interested in using these colors for the brand?

Reply:
Yeah that sounds great. I didn't change anything now but we can circle back and changes these soon.

> I love the axes! One case where they're a little less clear than the cube is where they are all going out in equal directions, so I tried out adding a little indigo sphere and kinda liked it, wanted to see your thoughts (see attached). Seeing the intersection point between the axis boxes and the sphere gave me a bit more orientation, so I could tell which side of the origin I was on if that makes sense.
 
Would you like to change the axis colors along with the sphere at the base of the axes?

>I think Deep Ocean and BeforeDawn could be good for building a brand around, Metallic has some character that fits with your current brand but idk if I would like that hot pink if there was a bunch of it in the model, hard to parse features looking at it.

Yup sweet, I've added deepOcean for now and changed the default script to add a `difference` so that some of the cut colour is present.